api/src/certification/shared/domain/models/Version.js

```diff
@@ -13,14 +13,20 @@ export class Version {
     scope: Joi.string()
       .required()
       .valid(...Object.values(Frameworks)),
-    challengesConfiguration: Joi.object().required(),
+    challengesConfiguration: Joi.object()
+      .keys({
+        defaultCandidateCapacity: Joi.number().required(),
+      })
+      .unknown(true)
+      .required(),
```

Comment:
🔖 **Reste à faire :** faire comme dans le domain de configuration et basculer le Joi sur `FlashAssessmentAlgorithmConfiguration`. Le faire va probablement nécessiter des adaptations dans certains tests.

Ici cette première étape (que j'avais aussi faite dans le domaine configuration) est de prouver au moins que `defaultCandidateCapacity` peut etre required.

```suggestion
    challengesConfiguration: Joi.object().instance(FlashAssessmentAlgorithmConfiguration).required(),,
```

Reply:
La bascule sera pour plus tard (dernière Pr du sujet).